Header header
time timecode
int16 angle_estimastion
uint8 angle_index
int16[] votes
int16[] supported_angles
